### PR TITLE
Created DBmarlin quickstart.

### PR DESCRIPTION
**Pull request by martinpinner**
# Summary

A concise description of the changes being introduced. If you are submitting a PR for a
new quickstart please review the pre merge checklist section to validate this PR is ready for review.

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you check your dashboard image quality? -  Do they look good?
- [ ] Did you check that your alerts actually work?
- [ ] Did you include an InstallPlan and Documentation reference?
- [ ] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [ ] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

Attach images of any visual changes, such as a dashboard here.

### CODE REVIEW

File: quickstarts/dbmarlin/config.yml
Comment (aswanson-nr) [inline quickstarts/dbmarlin/config.yml:51]: This install plan doesn't exist. If you're looking to install the infrastructure agent, can I suggest the [targeted infrastructure install plan](https://github.com/newrelic/newrelic-quickstarts/blob/main/install/infrastructure/infra-agent-targeted/install.yml)?

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
Hey @martinpinner! This looks fantastic, there's just one problem with the install plan and then we should be able to get this merged in.

File: quickstarts/dbmarlin/config.yml
Comment (jcountsNR) [inline quickstarts/dbmarlin/config.yml:11]:
```suggestion

```

This line is unnecessary, we'll want to add the integration doc under data sources.

File: quickstarts/dbmarlin/config.yml
Comment (jcountsNR) [inline quickstarts/dbmarlin/config.yml:56]:
```suggestion
    url: https://docs.dbmarlin.com/docs/integrations/newrelic
```

File: quickstarts/dbmarlin/config.yml
Comment (jcountsNR) [inline quickstarts/dbmarlin/config.yml:51]:
```suggestion
  - third-party-dbmarlin
```

File: quickstarts/dbmarlin/config.yml
Comment (jcountsNR) [inline quickstarts/dbmarlin/config.yml:13]:
```suggestion
  To get started with DBMarlin, visit the [DBmarlin introduction page](https://docs.dbmarlin.com/docs/DBmarlin/introduction).
```

**Review by jcountsNR — CHANGES_REQUESTED** [review]
@martinpinner , This looks very close, I just suggested a few changes to get this ready for merging. outside of the suggestions, you will need to add an install plan with id `third-party-dbmarlin`, and you will also need to change the dashboard image filename to `dbmarlin-dashboard01.png` so that it registers correctly. Hope this helps!

File: install/third-party/dbmarlin/install.yml
Comment (aswanson-nr) [inline install/third-party/dbmarlin/install.yml:13]: This is the link that user's will be sent to during installation, we might want to change it over to a link to documentation on how to setup the integration. The page at `https://www.dbmarlin.com/partners` makes no reference is such an integration.

File: quickstarts/dbmarlin/config.yml
Comment (aswanson-nr) [inline quickstarts/dbmarlin/config.yml:55]: This link is currently 404ing for me, does it need to be made public?

**Review by aswanson-nr — COMMENTED** [review]
Hey @martinpinner! Looks like @jcountsNR got you on the right track for the install plan and everything is looking good.
I just saw two issues with some of the links provided!

File: quickstarts/dbmarlin/config.yml
Comment (jcountsNR) [reply quickstarts/dbmarlin/config.yml:55]: @aswanson-nr  I don't think the docs are live yet, which is why the links aren't working. Should we wait on this PR until they are? Also, do you know why this is failing? I thought Icons were no longer used in quickstarts.

Comment (martinpinner) [reply quickstarts/dbmarlin/config.yml:55]: https://docs.dbmarlin.com/docs/integrations/newrelic is not live yet - it is still in staging. I hope to get it live in the next few days.

Comment (aswanson-nr) [reply quickstarts/dbmarlin/config.yml:55]: This will immediately be published after merging, so we should hold off on this PR until that documentation is live.

@jcountsNR, it looks like I missed that the dashboards are under the `quickstarts/dbmarlin` directory. Dashboards have been moved to their own top level directory to allow them to be shared between quickstarts. 
We'll just need to get this updated by moving the dashboard over to the `dashboards/` top level directory, then updating the `config.yml` to point to that dashboard. Full docs are available [here](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#quickstart-definitions)

Comment (aswanson-nr) [reply quickstarts/dbmarlin/config.yml:55]: Please let me know if either of your run into any issues!

Comment (martinpinner) [reply quickstarts/dbmarlin/config.yml:55]: https://docs.dbmarlin.com/docs/integrations/newrelic is now live.

Comment (jcountsNR) [reply quickstarts/dbmarlin/config.yml:55]: @aswanson-nr  ah you are right, I missed that one as well. @martinpinner  can you also move the dashboard to the correct directory? The docs are linked above.

**Comment by jcountsNR** [comment]
@aswanson-nr this looks like it's updated properly and the links are now active as well. Would you be able to take another look?

**Review by aswanson-nr — APPROVED** [review]
@martinpinner and @jcountsNR, looks good to go!

**Review by jcountsNR — APPROVED** [review]

**Pull request merged**
